src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/services/ccd/CreateCaseCallbackService.java

```diff
@@ -199,7 +173,7 @@ private ProcessResult tryCreateNewCase(
                 );
             } else if (ids.size() == 1) {
                 return new ProcessResult(
-                    prepareResultExceptionRecord(
+                    exceptionRecordFinalizer.finalizeExceptionRecord(
```

Comment:
Worth creating a local variable and set it with the result of `exceptionRecordFinalizer.finalizeExceptionRecord`. It wasn't clear to me what it returned. This is just a suggestion to consider.

Reply:
Will do